examples/data-sources/cluster/vcf_credentials.tf

```diff
@@ -1,15 +0,0 @@
-terraform {
-  required_providers {
-    vcf = {
-      source = "vmware/vcf"
-    }
-  }
-}
-
-data "vcf_credentials" "creds" {
-
-}
-
-data "vcf_credentials" "vcenter_credentials" {
-  resource_type = "VCENTER"
-}
```

Comment:
Why is this removed?

Reply:
this was a duplicate entry in the wrong folder. It's noted in the PR details.